File: deal/src/main/java/ru/artemev/deal/model/enums/Position.java
package ru.artemev.deal.model.enums;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;

@Schema(description = "Должность")
@AllArgsConstructor
public enum Position {
  WORKER("Рабочий"),
  MID_MANAGER("Менеджер среднего звена"),
  TOP_MANAGER("Топ-менеджер"),
  OWNER("Директор");

  private final String description;
}

File: gateway/src/main/kotlin/ru/artemev/gateway/model/enums/Position.kt
package ru.artemev.gateway.model.enums

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Должность")
enum class Position(val description: String) {
    WORKER("Рабочий"),
    MID_MANAGER("Менеджер среднего звена"),
    TOP_MANAGER("Топ-менеджер"),
    OWNER("Директор")
}